grants.yml

```diff
@@ -1776,6 +1780,7 @@
     - queue:create-task:highest:scriptworker-k8s/{trust_domain}-{level}-*
     - queue:create-task:highest:scriptworker-k8s/{trust_domain}-t-*
     - queue:create-task:highest:scriptworker-prov-v1/{trust_domain}-t-*
+    - queue:create-task:highest:scriptworker-prov-v1/{trust_domain}-signing-mac14m2
```

Comment:
You'll also need to remove this one. Without the `job:` key below, it'll default to everywhere which includes pull requests :)

Reply:
Mmmmm right. I've moved it down to main/release only